src/System.Management.Automation/engine/ArgumentTypeConverterAttribute.cs

```diff
@@ -152,11 +152,6 @@ internal object Transform(EngineIntrinsics engineIntrinsics, object inputData, b
                         if (_convertTypes[i] == typeof(ActionPreference))
```

Comment:
@KirkMunro, your last commit had 10 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

PowerShellGet - Script tests (Admin).Should install a script correctly to the required location with AllUsers scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1: line 234
234:         $installedScriptInfo | Should -Not -BeNullOrEmpty
```
Type inference Tests.Infers type from block statement
```powershell
Expected 'System.Int32', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Api/TypeInference.Tests.ps1: line 600
600:         $res.Name | Should -Be 'System.Int32'
```
PowerShellGet - Script tests.Should install a script correctly to the required location with default CurrentUser scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1: line 207
207:         $installedScriptInfo | Should -Not -BeNullOrEmpty
```
Tests for (error, warning, etc) action preference.ErrorAction = Suspend does not work on functions
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParameterBindingFailed,MyHelperFunction' to be thrown, but the FullyQualifiedErrorId was 'CannotConvertArgumentNoMessage,MyHelperFunction'. from /home/vsts/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1:75 char:45
    +             { MyHelperFunction -ErrorAction Suspend } | Should -Throw ...
    +                                             ~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 75
75:             { MyHelperFunction -ErrorAction Suspend } | Should -Throw -ErrorId "ParameterBindingFailed,MyHelperFunction"
```
Tests for (error, warning, etc) action preference.ErrorAction = Suspend does not work on cmdlets
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParameterBindingFailed,Microsoft.PowerShell.Commands.GetProcessCommand' to be thrown, but the FullyQualifiedErrorId was 'CannotConvertArgumentNoMessage,Microsoft.PowerShell.Commands.GetProcessCommand'. from /home/vsts/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1:79 char:40
    +             { Get-Process -ErrorAction Suspend } | Should -Throw -Err ...
    +                                        ~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 79
79:             { Get-Process -ErrorAction Suspend } | Should -Throw -ErrorId "ParameterBindingFailed,Microsoft.PowerShell.Commands.GetProcessCommand"
```